src/main/java/frc/robot/Robot.java

```diff
@@ -1,26 +1,40 @@
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.RunCommand;
 import frc.robot.subsystems.climber.ClimberSubsystem;
+import frc.robot.utils.PS4Constants;
 
 public class Robot extends TimedRobot {
 
-    
+    Joystick operatorControl = new Joystick(1);
+
+    int climbLeftAmnt = PS4Constants.LEFT_STICK_X.getId();
```

Comment:
I think these go in Buttons.java, and then we can access them by making a Buttons object in Robot.java